caffe-grid/src/main/python/examples/README-coco.md

```diff
@@ -0,0 +1,121 @@
+Steps to run the COCO dataset for Image Captioning
+==================================================
+##### (1) Env setup
+    export CAFFE_ON_SPARK=/Users/mridul/bigml/CaffeOnSpark
+    export DYLD_LIBRARY_PATH=${CAFFE_ON_SPARK}/caffe-public/distribute/lib:${CAFFE_ON_SPARK}/caffe-distri/distribute/lib
+    export DYLD_LIBRARY_PATH=${DYLD_LIBRARY_PATH}:/usr/local/cuda/lib:/usr/local/mkl/lib/intel64/
+    export LD_LIBRARY_PATH=${DYLD_LIBRARY_PATH}
+    export SPARK_HOME=/Users/mridul/bigml/spark-1.6.0-bin-hadoop2.6
+    export PATH=${SPARK_HOME}/bin:${PATH}
+    export PYSPARK_PYTHON=Python2.7.10/bin/python
+    export PYTHONPATH=$PYTHONPATH:caffeonsparkpythonapi.zip:caffe_on_grid_archive/lib64:/usr/local/cuda-7.5/lib64
+    export LD_LIBRARY_PATH=Python2.7.10/lib:/usr/local/cuda/lib:caffe_on_grid_archive/lib64/mkl/intel64/:${LD_LIBRARY_PATH}
+    export DYLD_LIBRARY_PATH=Python2.7.10/lib:/usr/local/cuda/lib:caffe_on_grid_archive/lib64/mkl/intel64/:${LD_LIBRARY_PATH}
```

Comment:
Why do we have 2 sets of definition for LD_LIBRARY_PATH and DYLD_LIBRARY_PATH. Please consolidate.